src/db.rs

```diff
@@ -1497,3 +1508,15 @@ fn snapshot_test() {
     let opts = Options::default();
     assert!(DB::destroy(&opts, path).is_ok());
 }
+
+#[test]
+fn set_option_test() {
+    let path = "_rust_rocksdb_set_optionstest";
+    {
+        let db = DB::open_default(path).unwrap();
+        assert!(db.set_option("disable_auto_compactions", "true").is_ok());
+        assert!(db.set_option("disable_auto_compactions", "false").is_ok());
+        assert!(db.set_option("disable_auto_compactions", "INVALID_VALUE").is_err());
```

Comment:
What about other options or this implementation related only for `disable_auto_compactions` option ?

Reply:
This should also work for other options as well (IIUC, the list is [here](https://github.com/facebook/rocksdb/blob/46e3209e0da1d7563e345d0e28a82ec05f1ddf32/options/options_helper.cc#L1663)). Currently in [electrs](https://github.com/romanz/electrs), I am planning using it for setting `disable_auto_compactions` to `false` after the initial bulk load is over.